examples/async-examples.js

```diff
@@ -0,0 +1,59 @@
+(async () => {	
+	const revai = require('../../revai-node-sdk');
+
+	/* Initialize your client with your revai access token */
+	const token = '<YOUR-ACCESS-TOKEN>';
+	var client = new revai.RevAiApiClient(token);
+
+	/* Get account details */
+	var account = await client.getAccount();
+	console.log(`Account: ${account.email}`);
+	console.log(`Balance: ${account.balance_seconds} seconds`);
+
+	
+	const jobOptions = {
+	    metadata: "This is an example",
```

Comment:
Let's use a more practical example.


```suggestion
	    metadata: "InternalOrderNumber=364",
```

Reply:
I think if you're going to have magic numbers, make sure they are well-known (people could get confused and think they are meaningful). e.g. `42`, `9001` are pretty good. I think best is to avoid any possible confusion.